javascript-data-model/order-history.js

```diff
@@ -7,11 +7,13 @@ var orderHistory = [
     details: '',
     invoice: '',
     delivered: 'Aug 8, 2020',
-    image: '',
-    title: 'Javascript for impatient programmers',
-    author: 'Rauschmayer, Dr. Axel',
-    return: 'Return window closed on Sep 7, 2020',
-    price: '31.55',
+    item: {
```

Comment:
Instead of making `item` an object that holds arrays, I would make the `item` property an array that holds an object for each item